velox/functions/sparksql/String.h

```diff
@@ -1556,4 +1556,73 @@ struct Empty2NullFunction {
   }
 };
 
+template <typename TOutString, typename TInString>
```

Comment:
Is it necessary to template this function, because the input and output type are fixed?